README.md

```diff
@@ -2,11 +2,11 @@
 
```

Comment:
As discussed i'd suggest splitting up the preparation readme and previously existing readme in 2. 
Maybe even make 3, where one is the readme.md that refers to both other readme's and has a nice overall intro of the goal and scope of this repo. SImilar to/including parts of what you've put here